src/Transformer/FractalTransformer.php

```diff
@@ -129,7 +131,10 @@ protected function createResource($response, $transformer, array $parameters)
      */
     public function parseFractalIncludes(Request $request)
     {
-        $includes = array_filter(explode($this->includeSeparator, $request->get($this->includeKey)));
+        $includes = $request->get($this->includeKey);
+
+        if(!is_array($includes))
```

Comment:
shorthand conditionals not permitted in PSR-2

https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md#51-if-elseif-else